### PR TITLE
Reusing components

### PR DESCRIPTION
**Pull request by S00185883**
Also changed some things looks wise and added the link to the workday
Screenshot 2021-08-05 at 15 41 59

### CODE REVIEW

File: src/components/popup/modal.js [hunk 1]
```diff
@@ -2,44 +2,65 @@ import React, { Component } from "react";
...
-          </Button>
-        </DialogActions>
-      </Dialog>
-    </>
-  )
+import '../PopUp/modal.css';
+
+
+class Modal extends Component {
+  componentDidMount() {
```
Comment (S00189802) [inline src/components/popup/modal.js]: We need to change this to a Functional Component

**Comment by S00189802** [comment]
Is the HTML for the Modal, inside the Header Component? We should probably have all the code for the modal inside the Modal Component  and then just call the Modal in the header.

File: src/components/popup/modal.js (hunk 1 above)
Comment (S00185883) [reply src/components/popup/modal.js]: cant change it me and Eimutis had a look earlier in the week

Comment (Clancinio) [reply src/components/popup/modal.js]: Your changes were merged along with Jamies last PR. Will explain later.

**Comment by S00185883** [comment]
I dont know why this was closed, it wasnt merged and nothing was added to the dev file

File: src/components/popup/modal.js (hunk 1 above)
Comment (Clancinio) [reply src/components/popup/modal.js]: Oh weird. Jamie though he had merge all of your changes because actually doesn't look like it

**Comment by Clancinio** [comment]
Make sure you are using the Prettier extension  and save on format. Doesn't look like your files are formatted

**Pull request closed without merging**